org-code-javabuilder/playground/src/main/java/org/code/playground/Item.java

```diff
@@ -0,0 +1,31 @@
+package org.code.playground;
+
+interface Item {
```

Comment:
there's been some discussion about changing this interface to a base class [link](https://docs.google.com/document/d/1Moo2s5EXZRp5rMg1VW9jlOqs_GeMN5yjU8FJgoqOEMk/edit?disco=AAAAPONopfg). Since discussion is still ongoing this could be done as a follow-up, but just an fyi

Reply:
I saw that :) I'm inclined to do as a follow-up to keep this moving if that sounds ok!